tests/components/default_config/test_init.py

```diff
@@ -34,4 +35,9 @@ async def test_setup(
 ) -> None:
     """Test setup."""
     recorder_helper.async_initialize_recorder(hass)
+    # default_config needs the homeassistant integration, assert it will be
+    # automatically setup by bootstrap and set it up manually for this test
+    assert "homeassistant" in bootstrap.CORE_INTEGRATIONS
+    assert await async_setup_component(hass, "homeassistant", {"foo": "bar"})
+
```

Comment:
This was not needed before this PR because the scene integration sets up the homeassistant integration here: https://github.com/home-assistant/core/blob/4a34cd25b2130afe254ae0500b552d1c65f63e9f/homeassistant/components/scene/__init__.py#L70-L71